src/config-schema.ts

```diff
@@ -90,6 +90,11 @@ export const configSchema = {
     _default: false,
     _description: 'Whether to enable form validation',
   },
+  enableRDESection: {
+    _type: Type.Boolean,
+    _default: false,
+    _description: 'Whether to enable the RDE section in the form builder',
```

Comment:
```suggestion
    _description: 'Whether to add an Encounter Details section for retrospective data entry, which includes encounter date/time and provider fields',
```

I think there's room for improving this config property description. 
```